functions/controller/Qa_QuestionDelete.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+
+class Qa_QuestionDelete
+{
+    private $db;
+
+    public function __construct()
+    {
+        require_once '../functions/datalayer/database.class.php';
+        $database = new Database();
+        $this->db = $database->getConnection();
+    }
```

Comment:
Zet de require_once boven je class. Anders doet ie elke keer de class includen terwijl het maar een keer nodig is.